defaults/main.yml

```diff
@@ -147,7 +147,10 @@ openio_oioswift_app_proxy_server:
   account_autocreate: true
   log_name: "OIO,{{ openio_oioswift_namespace }},oioswift,{{ openio_oioswift_serviceid }}"
   sds_chunk_checksum_algo: "md5"
-  #verb_acl = PUT,POST,PATCH,DELETE:127.,10.100.20.44
+
+openio_oioswift_verb_acl:
```

Comment:
the name must be `openio_oioswift_filter_verb_acl` if the filter's name is `verb_acl`